apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -838,7 +851,9 @@ export default connect(
     hidden: state.pageConstants.isShareView,
     shortInstructions: state.instructions.shortInstructions,
     isRtl: state.isRtl,
-    widgetMode: state.pageConstants.widgetMode
+    widgetMode: state.pageConstants.widgetMode,
+    dynamicInstructions: getDynamicInstructions(state.instructions), //state.instructions.dynamicInstructionsDefaults,
+    dynamicInstructionsKey: state.instructions.dynamicInstructionsKey
```

Comment:
do you need the defaults here?

Reply:
`getDynamicInstructions` takes the defaults and writes in the level-specific variants if there are any, providing the final desired set here.